src/lib/phdi-building-blocks/pyproject.toml

```diff
@@ -14,6 +14,10 @@ azure-storage-blob = "^12.12.0"
 polling = "^0.3.2"
 phonenumbers = "^8.12.48"
 pycountry = "^22.3.5"
+PyYAML = "^6.0"
+pyarrow = "^8.0.0"
+pandas = "^1.4.2"
```

Comment:
I don't recall seeing `pandas` used anywhere in here. Do we need it listed here?